### PR TITLE
Add ariana waller podcast

### PR DESCRIPTION
**Pull request by anjones1000**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-podcasts-screencasts-en.md [hunk 1]
```diff
@@ -240,7 +240,7 @@
 * [Toolsday](http://toolsday.io) (podcast)
 * [TTL Podcast](http://ttlpodcast.com) (podcast)
 * [Web Security Warriors](https://devchat.tv/web-security-warriors/) (podcast)
-
+* [Women in TECH with Ariana](https://podcasts.apple.com/us/podcast/women-in-tech-with-ariana/id1338917627) (podcast)
 
 ### PHP
```
Comment (kadhirash) [inline free-podcasts-screencasts-en.md:245]: Hello,
there needs to be an extra blank line (2 total) between the last resource and the next section.

Read about why [here](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)

```suggestion
* [Women in TECH with Ariana](https://podcasts.apple.com/us/podcast/women-in-tech-with-ariana/id1338917627) (podcast)


### PHP
```

Comment (anjones1000) [reply free-podcasts-screencasts-en.md:245]: Will do!

**Comment by kadhirash** [comment]
Build error now seems to be an error from the free-courses-en.md, something about a "Misplaced author" on line 315. 

Unable to see the error though.

**Comment by anjones1000** [comment]
I checked line 315 and there is a blank line before the Misc resource. I did not add a blank line to those resources and am unable to find the error. What can I do?

**Comment by SethiPandi** [comment]
https://angiejones.tech/free-java-course-test-automation is just a page that talks about and links to the course which is on another page; that page was already added to the repository in a previous PR. That would make this a duplicate that just adds page in the middle of this repo and the course.

I think that link can be removed from this PR.

**Comment by anjones1000** [comment]
Thank you, I'll remove it.

**Pull request merged**